docs/docs/reference/attestations.md

```diff
@@ -0,0 +1,193 @@
+---
+sidebar_position: 1
+title: Contract-less and auto-discovery of materials
+---
+A Workflow Contract specifies the necessary content that a workflow must include in its attestation. For instance, it might mandate 
+the inclusion of the URI@digest of the generated container image, the container root filesystem used during the build, and the 
+Software Bill of Materials for that container image. These pieces of evidence must be associated with a specific material type. 
+Operators define what must be included in the contracts, and developers need to understand and comply with these requirements.
```

Comment:
I'd add a couple of links when you talk about workflow contracts https://docs.chainloop.dev/reference/operator/contract